plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/FeatureDevSessionContext.kt

```diff
@@ -66,6 +66,198 @@ class FeatureDevSessionContext(val project: Project, val maxProjectSizeBytes: Lo
         "dist/"
     ).map { Regex(it) }
 
+    private val allowedCodeExtensions = setOf(
```

Comment:
nit - can we potentially load this from a json file so we don't have a long kt code?

Reply:
hm, potentially, however I do not see that as a pattern in this package

Reply:
yes, but I would argue that I don't see another large string like this one set in the package that requires extract to a file. Maybe would be good do for this one?

Reply:
+1, json or a constant file would make the actual code much readable.